### DISCUSSION TITLE
Made some pixelart based on supertux.

### DISCUSSION THREAD

**Discussion by Gustavekkj** [SuperTux/supertux#2264] [Show and tell]
A running scene.

![supertux running](https://user-images.githubusercontent.com/31554493/180801115-a8417822-cccc-4ae1-ac14-53fc798fa6c7.gif)

a new tux
![Sprite running](https://user-images.githubusercontent.com/31554493/180802376-d33e8cb8-7ec7-4f15-8a15-e7249389efae.gif)

a mini tux
![Sprite minitux](https://user-images.githubusercontent.com/31554493/180802098-fc01207b-f2dc-404c-955b-e60481ffd4cd.gif)
/Sprite%20running.gif

**Comment by CPa129** [comment]
Nice
